racon/1.4.3/Dockerfile

```diff
@@ -0,0 +1,47 @@
+# Use ubuntu as base image
+FROM ubuntu:xenial
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL version="1"
+LABEL software="Racon"
+LABEL software.version="1.4.3"
+LABEL description="Long read assembly and polishing tools"
+LABEL website="https://github.com/isovic/racon"
+LABEL license="See program repositories"
+LABEL maintainer="Erin Young"
+LABEL maintainer.email="eriny@utah.gov"
+
+# install python
+RUN apt-get update && apt-get install -y \
+   perl \
+   default-jre \
+   gnuplot \
+   libgcc-5-dev \
+   libgomp1 \
+   maven \
+   git \
+   wget \
+   python3 \
+   build-essential \
+   cmake \
+   zlib1g-dev && \
+   apt-get clean
```

Comment:
omit `apt-get clean` and replace with `apt-get autoclean && rm -rf /var/lib/apt/lists/*`